ci/cloudbuild/builds/lib/quickstart.sh

```diff
@@ -60,9 +60,14 @@ function quickstart::build_one_quickstart() {
     "-H${src_dir}"
     "-B${cmake_bin_dir}"
     -DCMAKE_PREFIX_PATH="${prefix}"
-    -DGOOGLE_CLOUD_CPP_ENABLE_CCACHE=ON
```

Comment:
Why remove in this case, but `=OFF` in others?

Reply:
Restored.  It should be removed because it goes unused (different top-level CMake file), but that belongs in a separate PR